src/lib/functions/form-submissions-handler.js

```diff
@@ -125,9 +125,9 @@ const createFormSubmissionHandler = function ({ getFunctionByName, siteUrl, warn
   }
 }
 
-const getFormHandler = function ({ getFunctionByName, warn }) {
+const getFormHandler = function ({ functionsRegistry, warn }) {
   const handlers = ['submission-created', `submission-created${BACKGROUND}`]
-    .map((name) => getFunctionByName(name))
+    .map((name) => functionsRegistry.get(name))
```

Comment:
The form submission handler needs to know whether a function with a given name is registered; this is now done on the function registry, and `getFunctionByName` is no longer a thing.